src/components/Message/Message.js

```diff
@@ -160,13 +155,15 @@ class Message extends Component {
     readBy: [],
     groupStyles: [],
     Attachment,
-    editing: false,
     messageActions: Object.keys(MESSAGE_ACTIONS),
   };
 
-  /** @type {(nextProps: Props) => boolean} Typescript syntax */
-  shouldComponentUpdate(nextProps) {
-    return shouldMessageComponentUpdate(nextProps, this.props);
+  /** @type {(nextProps: Props, nextState: State) => boolean} Typescript syntax */
+  shouldComponentUpdate(nextProps, nextState) {
+    return (
+      shouldMessageComponentUpdate(nextProps, this.props) ||
+      !deepequal(nextState, this.state)
```

Comment:
What you think of just having the explicit comparison here: `nextState.editing !== this.state.editing`?

Reply:
yea that makes a lot more sense

Reply:
actually let's keep this as it is since it might cause confusion in future